### ISSUE TITLE
Bug: 405 Method Not Allowed - visitor-keys

### ISSUE THREAD

**Issue by spo-pgraessle** [typescript-eslint/typescript-eslint#6430]
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have updated to the latest version of the packages.
- [X] I have [searched for related issues](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aissue+label%3Abug) and found none that matched my issue.
- [X] I have [read the FAQ](https://typescript-eslint.io/linting/troubleshooting) and my problem is not listed.

### Relevant Package

parser

### Repro Code

```typescript
From a clean environment, execute
`npm init`
`npm install @typescript-eslint/eslint-plugin`
```

### Expected Result

I would have expected the package to install.  I tried to use a different version 5.50.0; however, when attempting to install that version I received the same message.

### Actual Result

405 error was encountered.

npm ERR! code E405
npm ERR! 405 Method Not Allowed - GET https://npm.pkg.github.com/ruelala/@typescript-eslint/visitor-keys/-/visitor-keys-5.51.0.tgz

### Additional Info

If I add `@typescript-eslint/parser` as a dependency in package.json, it will work for **5.50.0** and continue to fail with the error mentioned above for **5.51.0**

### Versions

| package                                | version |
| -------------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin`     | `5.51.0` |
| `@typescript-eslint/parser`            | `5.51.0` |
| `node`                                 | `18.13.0` |

**Comment by bradzacher** [comment]
That is something to do with npm's servers.
https://www.npmjs.com/package/@typescript-eslint/visitor-keys
The package is published according to npm's website.
https://unpkg.com/browse/@typescript-eslint/visitor-keys@5.51.0/
Unpkg which pulls from npm can see 5.51.0

Please be patient and try again.
